### ISSUE TITLE
Identical gcmc formation energies at different temperatures?

### ISSUE THREAD

**Issue by DS-Wen** [prisms-center/CASMcode#189]
Hi,

I was using CASM to study a pseudobinary system Ni3(Al,Ta), varying Al and Ta concentrations on the sublattice. My cluster expansion seems good but I find that the gcmc formation energies for 873 K and 1173 K are almost identical. Both gcmc formation energies have some portions below the cluster expansion convex hull. I wonder what could cause this problem?

Thanks,
Dongsheng 

![image](https://user-images.githubusercontent.com/33857502/92077268-a38abf80-edee-11ea-9d8a-46985bbc47cf.png)

**Comment by bpuchala** [comment]
The `clex_hull` values are calculated from existing configurations in your project. The monte carlo calculation may encounter different configurations with lower clex formation energies.

**Comment by DS-Wen** [comment]
Thank you! 
I found this post #149 and it seems I have a similar situation that my CE predicts low energy configurations that are not in the enumerated pool. My MC or GCMC supercells contain more than 1000 atoms and it is impossible to calculate such large structures. My concern is how to obtain the ground-state configurations outside my enumerated configuration pool?

My primitive supercell contains 8 distinct-sites to enumerate. And more than 3 millions of configuration will be generated if I go up to 3x volume supercells. The config.json is too large to read in this case. So I tried to reduce the number of configurations by restricting the maximum number (less than 100,000). Now I realize perhaps configurations with larger volumes can have energy way below the current convex hull. 

Now I'm thinking if I could restrict the MC cell and try to predict the LT configurations with 4x, 5x, or 6x volume? And how do I add the new configurations to the project in order to calculate them? 

Best,
Dongsheng
